src/github.com/segmentio/go-loggly

```diff
@@ -0,0 +1 @@
+Subproject commit e78f6971ebca5835614673e2f5f6a47ca5f13501
```

Comment:
We don't want submodules.  Did you `go get` these?  If so, please delete and use `gost get` instead.  See the [Gost README](https://github.com/getlantern/gost) for information.